extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php

```diff
@@ -16,6 +16,34 @@ class CommunityPageSpecialUsersModel {
 
 	const ALL_CONTRIBUTORS_MODAL_LIMIT = 50;
 
+	// order of permissions is consistent with badges hierarchy in Discussions
+	const PERMISSION_HIERARCHY = [
+		'sysop',
+		'threadmoderator',
+		'content-moderator',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "content-moderator" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)